### DISCUSSION TITLE
Custom block missing

### DISCUSSION THREAD

**Discussion by subhash16** [Sketchware-Pro/Sketchware-Pro#922] [Ideas]
If I use custom block & somehow I accidently delete the custom block from block manager then If I build the current project then the project build successfully without any errors or notifying me.But it's showing missing in the used custom block section.
Please do a dialogue or anything when we open the project if there is missing block.
![IMG_20230723_234124](https://github.com/Sketchware-Pro/Sketchware-Pro/assets/37414496/aca317b0-7a15-4e9a-bac3-93eef478d12f)

**Comment by subhash16** [comment]
Please take a look into this matter

**Reply by j-px** [reply]
i already have an idea how to implement this. It'd be paired with improving AdMob, by the way
